### PR TITLE
new functionality for nmea transform

### PR DESCRIPTION
**Pull request by ellapiet**
add choice for which nmeas to use, add dpt and stn nmeas, add test

### CODE REVIEW

File: logger/transforms/nmea_transform.py [hunk 1]
```diff
@@ -40,8 +41,21 @@ class NMEATransform:
...
+                List of the nmea transforms that will be used.
+        **kwargs
+                Arugments needed for the nmea transforms, see transforms below for what will be included.
+        """
+        class_module_name = 'logger.transforms.nmea_transform'
+        module = importlib.import_module(class_module_name)
+
+        self.transforms = []
+        for transform in nmea_list:
+            class_const = getattr(module, transform)
```
Comment (davidpablocohn) [inline logger/transforms/nmea_transform.py:56]: Sanity checking here - can we fail more gently if user specifies a transform that isn't defined? I'm guessing it's too messy to procedurally emit a message that says what the allowable transforms are, but perhaps we could refer them to the code?

**Review by davidpablocohn — CHANGES_REQUESTED** [review]
I've not gone into depth on the the new transforms themselves, but in general they look reasonable. Only change I'd like would be some guardrails if the user specifies a transform that doesn't exist, so they can pinpoint what's wrong with their config.

Thanks!

File: logger/transforms/nmea_transform.py (hunk 1 above)
Comment (ellapiet) [reply logger/transforms/nmea_transform.py]: Done! Added check to compare provided transforms with all classes from the nmea_transforms file

File: logger/transforms/nmea_transform.py [hunk 2]
```diff
@@ -23,6 +23,7 @@
 # flake8: noqa E501  - ignore long comment lines that describe formats
 
 import logging
+import importlib
```
Comment (ellapiet) [inline logger/transforms/nmea_transform.py:27]: `import inspect`

**Review by davidpablocohn — APPROVED** [review]

**Pull request merged**